src/posix.mak

```diff
@@ -532,8 +532,8 @@ $G/dmd.conf: $(SRC_MAKE)
 ######## optabgen generates some source
 optabgen_output = debtab.d optab.d cdxxx.d elxxx.d fltables.d tytab.d
 
-$G/optabgen: $C/optabgen.c $C/cc.h $C/oper.h
-	$(HOST_CXX) $(CXXFLAGS) -I$(TK) $< -o $G/optabgen
+$G/optabgen: $C/optabgen.d $C/cc.d $C/oper.d
```

Comment:
You need to depend on the host compiler whenever compiling a D file. I'm currently on my phone (the other targets like `dmd` should already depend on the host compiler).
The reason is that otherwise the makefile can't do the auto bootstrapping.